x-pack/plugins/ml/public/explorer/explorer.js

```diff
@@ -900,10 +907,44 @@ export const Explorer = injectI18n(injectObservablesAsProps(
       }
     }
 
+    applyFilterFromTable = (fieldName, fieldValue, action) => {
+      let newQueryString = '';
+      const { queryString } = this.state;
+      const operator = 'and ';
+
+      if (action === '+') {
```

Comment:
Suggest to move`'+'` and `'-'` to an enum in `explorer_constants.js`, something like 

```javascript
const FILTER_ACTION = {
  ADD: '+',
  REMOVE: '-',
};
```

Reply:
Good idea! 👍